src/server/server.php

```diff
@@ -339,14 +339,34 @@
 		$text = preg_replace(array('/^\s*/', '/\s*$/', '/\s+/'), array('', '', ' '), optional_param('text', '', PARAM_TEXT));
 
 		// Compare strings ignoring case
-		// TODO: insert here the use of the Levenshtein distance
-		if (mb_strtolower($text, 'UTF-8') == mb_strtolower($elements[$number]['content'], 'UTF-8')
-			|| $options['usetransliteration']
-			&& strtolower(@iconv('UTF-8', 'ASCII//TRANSLIT', $text)) == strtolower(@iconv('UTF-8', 'ASCII//TRANSLIT', $elements[$number]['content'])))
+		if (false == $options['usecasesensitive'])
+		{
+			$parsed_text = mb_strtolower($text, 'UTF-8');
+			$parsed_content = mb_strtolower($elements[$number]['content'], 'UTF-8');
+		}
+		else
+		{
+			$parsed_text = $text;
+			$parsed_content = $elements[$number]['content'];
+		}
+
+		$previous_locale = setlocale(LC_ALL, 0);
+
+		if (false === setlocale(LC_ALL, str_replace("-",  "_", $options["language"])))
```

Comment:
For linux, a way is to test different value for the locale:
~~~
setlocale(LC_ALL, 'fr_FR.UTF8', 'fr.UTF8', 'fr_FR.UTF-8', 'fr.UTF-8')
~~~
See
http://php.net/manual/fr/function.setlocale.php#87038